### PR TITLE
bazel_5: 5.4.0->5.4.1

### PR DESCRIPTION
**Pull request by dmivankov**
Also update the updater script.

https://github.com/bazelbuild/bazel/releases/tag/5.4.1

- [X] recalculate fetchAttrs digest for packages built with bazel_5

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by dmivankov** [comment]
@ofborg build python3Packages.tensorflow
@ofborg build python3Packages.tensorflowWithCuda
@ofborg build envoy
@ofborg build bazel-watcher

**Comment by dmivankov** [comment]
@ofborg build python3Packages.jaxlib

**Comment by dmivankov** [comment]
@ofborg build python3Packages.tensorflow
@ofborg build python3Packages.tensorflowWithCuda
@ofborg build envoy
@ofborg build bazel-watcher
@ofborg build python3Packages.jaxlib

**Comment by dmivankov** [comment]
@ofborg build python3Packages.tensorflow
@ofborg build python3Packages.tensorflowWithCuda
@ofborg build envoy
@ofborg build bazel-watcher
@ofborg build python3Packages.jaxlib

File: pkgs/build-support/build-bazel-package/default.nix [hunk 1]
```diff
@@ -67,7 +67,6 @@ let
         --output_user_root="$bazelUserRoot" \
         ${cmd} \
         --curses=no \
-        -j $NIX_BUILD_CORES \
```
Comment (dmivankov) [inline pkgs/build-support/build-bazel-package/default.nix:70]: https://github.com/NixOS/nixpkgs/pull/227432/checks?check_run_id=13035938483
aarch64-linux build of bazel-watcher worked with old code but x86_64 was failing without this change :thinking: https://github.com/NixOS/nixpkgs/pull/227432/checks?check_run_id=13035942786
```
       > no configure script, doing nothing
       > building
       > patching script interpreter paths in .
       > ./release/release.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/zlf0f88vj30sc7567b80l52d19pbdmy2-bash-5.2-p15/bin/bash"
       > ./release/npm/package.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/zlf0f88vj30sc7567b80l52d19pbdmy2-bash-5.2-p15/bin/bash"
       > ./tools/workplace_status.py: interpreter directive changed from "#! /usr/bin/env python3" to "/nix/store/fdqpyj613dr0v1l1lrzqhzay7sk4xg87-python3-3.10.10/bin/python3"
       > Extracting Bazel installation...
       > ERROR: -j :: Unrecognized option: -j
       For full logs, run 'nix log /nix/store/8db0365qwz8j1fdrp5d0fb5hajjhxp5j-bazel-watcher-0.21.2-deps.tar.gz.drv'.
error: 1 dependencies of derivation '/nix/store/grg2xai3g2gzkk7vr0crgqagsqjm9ljl-bazel-watcher-0.21.2.drv' failed to build
```

**Comment by dmivankov** [comment]
@ofborg build python3Packages.tensorflow
@ofborg build python3Packages.tensorflowWithCuda
@ofborg build envoy
@ofborg build bazel-watcher
@ofborg build python3Packages.jaxlib
@ofborg build python3Packages.jaxlibWithCuda

**Review by uri-canva — CHANGES_REQUESTED** [review]
Back to you to fix the `-j` handling. Also change it to `--jobs` so it's easier to find in bazel code / documentation.

**Comment by dmivankov** [comment]
@ofborg build python3Packages.tensorflow
@ofborg build python3Packages.tensorflowWithCuda
@ofborg build envoy
@ofborg build bazel-watcher
@ofborg build python3Packages.jaxlib
@ofborg build python3Packages.jaxlibWithCuda

**Comment by dmivankov** [comment]
got distracted from this PR, but now rebased and update with new hashes and --jobs

**Review by uri-canva — APPROVED** [review]

**Pull request merged**